public/src/world/projectile.js

```
@@ -1,18 +1,22 @@
 import * as THREE from 'three';
-import { Projectile as ProjectileConst } from '../utils/constants.js';
 
 export class Projectile {
-    constructor(startPosition, direction) {
+    constructor(startPosition, direction, game) {
+        const skillData = game.data.skills?.shockwave; // Add defensive check
+        if (!skillData) {
+            console.warn('Skill data for shockwave not found. Using default values.');
+            skillData = { FP_COST: 20, DURATION: 1000, SPEED: 10, LIFESPAN: 2, DAMAGE: 30 };
```

Biome (1.9.4)

[error] Can't assign skillData because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)